File: backend/src/main/java/magazyn/dto/PanelAdminaDTO.java
package magazyn.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PanelAdminaDTO {

    private Integer liczbaUzytkownikow;
    private Integer liczbaProduktu;
    private Integer liczbaMagazynow;
    private Integer liczbaDostaw;
    
    private BigDecimal przychodyMiesiac;
    private BigDecimal wydatkiMiesiac;
    private BigDecimal zyskMiesiac;
    
    private Integer liczbaZamowienWProgress;
    private Integer liczbaZamowienDoRealizacji;
    
    private Integer liczbaProductowPonizejProgu;
}

File: frontend/app/src/main/java/com/example/magazyn/api/dtos/PanelAdminaDTO.kt
package com.example.magazyn.api.dtos

import java.math.BigDecimal

data class PanelAdminaDTO(
    val liczbaUzytkownikow: Int = 0,
    val liczbaProduktu: Int = 0,
    val liczbaMagazynow: Int = 0,
    val liczbaDostawow: Int = 0,
    val przychodyMiesiac: BigDecimal = BigDecimal.ZERO,
    val wydatkiMiesiac: BigDecimal = BigDecimal.ZERO,
    val zyskMiesiac: BigDecimal = BigDecimal.ZERO,
    val liczbaZamowienWProgress: Int = 0,
    val liczbaZamowienDoRealizacji: Int = 0,
    val liczbaProductowPonizejProgu: Int = 0
)
